internal/appsec/waf_test.go

```diff
@@ -253,11 +264,24 @@ func TestBlocking(t *testing.T) {
 			status:    403,
 			ruleMatch: ipBlockingRule,
 		},
+		{
+			name:     "body/no-block",
+			endpoint: "/body",
+			status:   200,
+			reqBody:  "Happy body existing",
+		},
+		{
+			name:      "body/block",
+			endpoint:  "/body",
+			status:    403,
+			reqBody:   "$globals",
+			ruleMatch: bodyBlockingRule,
+		},
 	} {
 		t.Run(tc.name, func(t *testing.T) {
 			mt := mocktracer.Start()
 			defer mt.Stop()
-			req, err := http.NewRequest("POST", srv.URL+tc.endpoint, nil)
+			req, err := http.NewRequest("POST", srv.URL+tc.endpoint, strings.NewReader(tc.reqBody))
```

Comment:
You can add an additional check on the response body to make sure it changes accordingly when blocking/not blocking

Reply:
Isn't it done [here](https://github.com/DataDog/dd-trace-go/blob/9593c685906779f762f2999dc8315d375654a9c1/internal/appsec/waf_test.go#L299) ?

Reply:
My mistake 👍